From 4fa6908969fa3440e55611fd15bdd973787f3c9d Mon Sep 17 00:00:00 2001
From: Aleksander Gondek <gondekaleksander@protonmail.com>
Date: Wed, 4 Feb 2026 20:58:16 +0100
Subject: [PATCH] patch: cc_shared_library_info_bzl made public

This changeset modifies the visibility of bzl_library
//cc/common:cc_shared_library_info_bzl so that
it can be accessed by ruleset clients.

Why its not public is beyond me.
---
 cc/common/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cc/common/BUILD b/cc/common/BUILD
index d564807..9626ee7 100644
--- a/cc/common/BUILD
+++ b/cc/common/BUILD
@@ -51,7 +51,7 @@ bzl_library(
 bzl_library(
     name = "cc_shared_library_info_bzl",
     srcs = ["cc_shared_library_info.bzl"],
-    visibility = ["//cc:__subpackages__"],
+    visibility = ["//visibility:public"],
     deps = [
         "@cc_compatibility_proxy//:symbols_bzl",
     ],
-- 
2.51.2

